metadata-db-app/int_test/cmr/metadata_db/int_test/concepts/force_delete_test.clj

```diff
@@ -66,3 +75,82 @@
        ;; tag association associated with other collection revision or whole collection is not deleted
        (util/is-tag-association-deleted? ta1 false)
        (util/is-tag-association-deleted? ta3 false)))))
+
+(deftest force-delete-service
+  (let [coll (util/create-and-save-collection "REG_PROV" 1)
+        coll-concept-id (:concept-id coll)
+        misc-svc (util/create-and-save-service "REG_PROV" 42)
+        misc-svc-assn (util/create-and-save-service-association
+                       coll misc-svc 1)
+        expected-svc-concept-id "S1200000003-REG_PROV"
+        expected-revision-ids [1 2 3]
+        revision-count 3]
+    (testing "delete most recent revision"
+      (let [latest-revision (util/create-and-save-service
+                             "REG_PROV" 1 revision-count)
+            concept-id (:concept-id latest-revision)
+            svc-assn (util/create-and-save-service-association
+                      coll latest-revision 1)]
+        (testing "initial conditions"
+          ;; creation results as expected
+          (is (= expected-svc-concept-id concept-id))
+          (is (= (last expected-revision-ids) (:revision-id latest-revision)))
+          ;; service revisions in db
+          (is (= 4
```

Comment:
I'm having trouble following the test setup. I see two calls to create services above - why are there 4 revisions? I see a call to:

`(util/create-and-save-service "REG_PROV" 1 revision-count)`

Which returns a revision - is this actually creating 3 revisions and returning only one?

Reply:
Yup.

Reply:
🤢